doc/lua_api.md

```diff
@@ -1678,10 +1678,12 @@ type are ignored.
 
 Displays an image on the HUD.
 
-* `scale`: The scale of the image, with 1 being the original texture size.
-  Only the X coordinate scale is used (positive values).
-  Negative values represent that percentage of the screen it
-  should take; e.g. `x=-100` means 100% (width).
+* `scale`: The scale of the image, with `{x = 1, y = 1}` being the original texture size.
```

Comment:
The CSM API has the same documentation flaw. I think it would be easier to just refer to lua_api.md to avoid duplications.

Reply:
I think I'll make a separate PR for removing all the outdated duplications from the CSM API; formspecs and particle spawners for example are also concerned.